lib/cartopy/crs.py

```diff
@@ -3219,6 +3219,59 @@ def x_limits(self):
     def y_limits(self):
         return self._y_limits
 
+class Spilhaus(Projection):
+    """
+    Spihaus-Adams World Ocean Map in a Square II
+
+    """
+    def __init__(self,greenland_at = 0,false_easting = 0.0,false_northing = 0.0):
```

Comment:
We don't have any other projections with "location_at" I don't think. I don't think we want that since it doesn't really have a direct relevance to the underlying projection. I think you can expose the `rotation=45` parameter here and pipe that through for rotation directly and let users handle the "greenland_at" logic with a wrapper function if they want that.

Do we want `central_longitude` and `central_latitude` to be exposed here too? They have different default values than typical though, so we could just leave them as default and not let the user have the option.

Note the formatting/linting of the arguments here too, there should be a space after a comma.

Reply:
I have now fully exposed the rotation parameter. I don't entirely understand why exposing central_lat lon "not let the user have the option", but I have also exposed those two variables.

Reply:
I ended up deciding not to expose central_longitude and central_latitude.